.tool-versions

```diff
@@ -0,0 +1,2 @@
+elixir 1.9.1
```

Comment:
As this is a library that is meant to support multiple Elixir versions, can we ignore the `.tool-versions` file?